filebeat/module/system/auth/_meta/fields.yml

```diff
@@ -24,6 +24,87 @@
       description: >
         The Unix user that this event refers to.
 
+    - name: pam
+      type: group
+      description: >
+        Fields specific to PAM messages.
+      fields:
+      - name: module
+        description: >
+          The name of the PAM module.
+      - name: service
+        description: >
+          The name of the PAM service.
+      - name: type
+        description: >
+          The PAM type.
+      - name: message
+        type: text
+        description: >
+          The PAM message text for otherwise unparsed events.
+      - name: result
+        type: text
+        description: >
+          The PAM result text, e.g. "authentication success".
+      - name: euid
+        type: long
+        description: >
+          The EUID of the PAM process.
+      - name: user
+        description: >
+          The username.
+      - name: auth
+        type: group
+        description: >
+          Fields specific to PAM auth messages.
+        fields:
+        - name: logname
+          description: >
+            The user initiating the PAM auth call.
+        - name: uid
+          type: long
+          description: >
+            The UID of the PAM auth process.
+        - name: euid
+          type: long
+          description: >
+            The EUID of the PAM auth process.
+        - name: tty
+          description: >
+            The TTY of the PAM auth process.
+        - name: ruser
+          description: >
+            The requesting user name.
+        - name: rhost
+          description: >
+            The requesting hostname or IP address.
```

Comment:
`remote.user` and `remote.host` instead of `ruser` and `rhost`? Just proposing, I am ok also with `ruser` and `rhost` if they are usual names for these values in the PAM context :slightly_smiling_face: